lib/controller.js

```diff
@@ -92,7 +95,7 @@ Tessel.list = function(opts) {
 
 Tessel.get = function(opts) {
   return new Promise(function(resolve, reject) {
-    logs.info('Looking for your Tessel...');
+    logs.info('Searching nearby LAN (netmask) and USB for Tessels...');
```

Comment:
@johnnyman727 as mentioned I opened an issue https://github.com/tessel/t2-cli/issues/340 for further discussions.